.gitignore

```diff
@@ -3,3 +3,4 @@ config
 tou
```

Comment:
this needs to be taken out. joseph made a pull request with this deleted so i think you just need to merge those changes to this branch